### DISCUSSION TITLE
running tests against selenoid "grid" and adding "selenoid:options" capability

### DISCUSSION THREAD

**Discussion by webmoskal** [seleniumbase/SeleniumBase#1087] [Q&A] [answered]
Can't figure out how to make desired_cap work with selenoid.  Selenoid expects `"selenoid:options", {"enableVNC": True}`

Tried: pytest [test] --server=http://localhost:4444/wd/hub --chrome --cap-string='{"selenoid:options": {"enableVNC": true}}'  `True` triggers json.loads error

So far only able to make it work by adding ` chrome_options.set_capability("selenoid:options", {"enableVNC": True})` to seleniumbase>core>browser_launcher.py>get_remote_driver

```
        if selenium4:
            chrome_options.set_capability("cloud:options", capabilities)
            chrome_options.set_capability("selenoid:options", {"enableVNC": True})
            return webdriver.Remote(
                command_executor=address,
                options=chrome_options,
                keep_alive=True,
            )
```

**Comment by mdmintz** [comment]
@webmoskal  I created a ticket for this and will have a new release soon: https://github.com/seleniumbase/SeleniumBase/issues/1088

**Comment by mdmintz** [comment, marked as answer]
@webmoskal This has been resolved by https://github.com/seleniumbase/SeleniumBase/releases/tag/v2.2.2
* On the command-line, add ``--cap-string='{"selenoid": "true"}'``
* Or use ``--cap-file=CAP_FILE.py`` with a line that contains:
    ``"selenoid": "true",``
    https://github.com/seleniumbase/SeleniumBase/blob/master/examples/capabilities/selenoid_cap_file.py

(For minimum SeleniumBase version ``2.2.2``.)
